src/main/java/com/github/alex1304/jdash/client/AuthenticatedGDClient.java

```diff
@@ -120,6 +123,23 @@ void putExtraParams(Map<String, String> params) {
 		return fetch(new GDMessageSendRequest(this, recipientAccountId, subject, body));
 	}
 
+	/**
+	 * Gets ranking of user with specific strategy, count.
+	 *
+	 * @param strategy the strategy of user ranking
+	 * @param count the number of users in the ranking to get
+	 * @return a Mono emitting a ranking list containing all users matching strategy.
+	 *		   To further explain the {@param count}, count=100 for Top100, creators list
+	 *		   count=50 for friends, global list. Exceptionally, a maximum about 1500 to 2,000
+	 *		   count can be used in for creators strategy.
+	 *
+	 * @throws UnsupportedOperationException if this client is not logged it to any
+	 *                                       account
+	 */
+	public Mono<List<GDUserSearchData>> getUserRanking(UserRankingStrategy strategy, int count){
```

Comment:
I find "user ranking" a weird name for something that simply fetches leaderboards. Why not just call this `getLeaderboard`? User ranking could make people think it just gets your global rank (so returns an integer and not a list of users). I was myself confused at first, so I think it would be better to use "leaderboard" instead of "user ranking" (so not only this method's name should change but also the name of the request classes and anything else mentioning "user ranking")

Reply:
Changed to _getLeaderboard_